logoapp/upload.php

```diff
@@ -1,10 +1,76 @@
 <?php
-echo '<pre>';
-echo $_POST["image"];
+echo "処理中です…<br>";
 
-if(move_uploaded_file($_POST["image"], "upload/test.jpg")){
+require('set_db.php');
+$id = "NuLl";
+
+//echo '<pre>';
+//print_r($_POST);
+
+$image = $_POST["image"];
+//echo "image: " .$image;
+//echo '<br>';
+$data = $_POST["data"];
+//echo "data: " .$data;
+//echo '<br><br>';
+$type = $_POST["type"];
+//echo "type: ".$type;
+//echo '<br><br>';
+
+if(strcmp($type,"dot") == 0){//dotかlineかを確認
+  //データベースにデータを登録
+  if(mysqli_query($link, "INSERT INTO `dot` (`ID`, `date`, `imgData`) VALUES (null, CURRENT_TIMESTAMP, '$data')") == TRUE){
+    //最新idを取得
+    $id = mysqli_fetch_array(mysqli_query($link, "SELECT * FROM `dot` ORDER BY `ID` DESC"))['ID'];
+    //echo "id: $id";
+  }else{
+    //echo "database error";
+  }
+}else if(strcmp($type,"line") == 0){
+  if(mysqli_query($link, "INSERT INTO `line` (`ID`, `date`, `imgData`) VALUES (null, CURRENT_TIMESTAMP, '$data')") == TRUE){
+    //最新idを取得
+    $id = mysqli_fetch_array(mysqli_query($link, "SELECT * FROM `line` ORDER BY `ID` DESC"))['ID'];
+    //echo "id: $id";
+  }else{
+    //echo "database error";
+  }
+}else{
+  //echo "file select error";
+}
+
+/*
+if($file_get_contents = file_get_contents($_POST["data"])){
+  file_put_contents('/upload/test.jpg', $file_get_contents);
   echo "<br>uploaded";
   exit();
 }else{
   echo "<br>upload error";
-}
\ No newline at end of file
+}
+*/
+
+//$file_get_contents = file_get_contents($data);
+//echo $file_get_contents;
+
+$contents_split = explode(',', $image);
+$encoded = $contents_split[count($contents_split)-1];
+$decoded = "";
+$fileName= $id;
+
+for ($i=0; $i < ceil(strlen($encoded)/256); $i++) {
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected at least 4 spaces, found 2
* No space found after comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->